editions/tw5.com/tiddlers/filters/syntax/Interchangeable Filter Run Prefixes.tid

```diff
@@ -0,0 +1,44 @@
+created: 20230316151518640
+modified: 20230327130626715
+tags: [[Filter Run Prefix]]
+title: Interchangeable Filter Run Prefixs
```

Comment:
Should be "prefixes".

I'm not sure why the word "Interchangeable" is needed here?

Reply:
Yea. That's a bit tricky. I did wanted to have that content in a separate tiddler and that was the best name I could come up with. 

 We do have [Filter Run Prefix](https://tiddlywiki5-ob4l4ycdk-jermolene.vercel.app/#Filter%20Run%20Prefix) which links to 

![image](https://github.com/Jermolene/TiddlyWiki5/assets/374655/e45c075a-2f70-4b2c-bd8e-baf8ae08e24e)

- [Named Filter Run Prefix](https://tiddlywiki5-ob4l4ycdk-jermolene.vercel.app/#Named%20Filter%20Run%20Prefix)
  - The named prefixes only **link** to the Interchangeable info, because I do not want to have the shortcuts so prominent. IMO usese should use the named prefixes
- [Shortcut Filter Run Prefix](https://tiddlywiki5-ob4l4ycdk-jermolene.vercel.app/#Shortcut%20Filter%20Run%20Prefix)
   - The shortcut prefix **transcludes** [Interchangeable Filter Run Prefixes](https://tiddlywiki5-ob4l4ycdk-jermolene.vercel.app/#Interchangeable%20Filter%20Run%20Prefixes) because I wanted to point out the named prefixes

Improvements are very welcome here.